src/conf_mode/policy-local-route.py

```diff
@@ -93,36 +117,54 @@ def apply(pbr):
         return None
 
     # Delete old rule if needed
-    if 'rule_remove' in pbr:
-        for rule in pbr['rule_remove']:
-            if 'source' in pbr['rule_remove'][rule]:
-                for src in pbr['rule_remove'][rule]['source']:
-                    call(f'ip rule del prio {rule} from {src}')
-            if 'fwmark' in  pbr['rule_remove'][rule]:
-                for fwmk in pbr['rule_remove'][rule]['fwmark']:
-                    call(f'ip rule del prio {rule} from all fwmark {fwmk}')
+    for rule_rm in ['rule_remove', 'rule6_remove']:
+        if rule_rm in pbr:
+            v6 = " -6" if rule_rm == 'rule6_remove' else ""
+            for rule in pbr[rule_rm]:
+                sources = ['']
+                if 'source' in pbr[rule_rm][rule]:
```

Comment:
Please add a bail out early statement:

```python
if 'source' not in pbr[rule_rm][rule]:
    continue
```

Reply:
This won't work, I want to write a rule even if there is no from/to/fwmark. It should be enough to have one of those set to a value to write a route. In that case, we would iterate over the empty element and not write a source, but maybe serveral destinations.

Maybe something like `for src in (pbr[rule_rm][rule]['source'] or ['']` would work? I'm not sure how that behaves if the rule has no `'source'` field.

Reply:
… or [] will work, used it here and there in the codebase